qcschema/dev/wavefunction/scf_wavefunction.py

```diff
@@ -0,0 +1,78 @@
+"""
+The complete list of SCF level wavefunction quantities.
+"""
+
+scf_wavefunction = {}
+
+# Orbitals
+scf_wavefunction["scf_orbitals_a"] = {
+    "type": "array",
+    "description": "SCF alpha-spin orbitals",
+    "items": {"type": "number"}
+}
+
+
+scf_wavefunction["scf_orbitals_b"] = {
+    "type": "array",
+    "description": "SCF beta-spin orbitals",
+    "items": {"type": "number"}
+}
+
+# Density
+scf_wavefunction["scf_density_a"] = {
+    "type": "array",
+    "description": "SCF alpha-spin density",
+    "items": {"type": "number"}
+}
+
+
+scf_wavefunction["scf_density_b"] = {
+    "type": "array",
+    "description": "SCF beta-spin density",
+    "items": {"type": "number"}
+}
+
+
+# Fock matrix
+scf_wavefunction["scf_fock_a"] = {
+    "type": "array",
```

Comment:
Is this Fock matrix meant to be in the AO basis or the MO basis? If it is in the AO basis do we need to store the Fock matrix in the localized MO basis separately (if localized orbitals were used)?

Reply:
Good catch. All matrices are in the AO basis, except for the LMO fock matrix where there is information to be gained from having the fock matrix in the LMO basis. Descriptions have been updated accordingly.